charts/kcale/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: kscale
 description: A Helm chart for kscale deployment
 type: application
-version: 0.0.1
+version: 0.0.2
```

Comment:
This isn't necessary because the action will automatically replace this with the correct value.

Reply:
it was the linting that wanted me to do it 😮 
I have reverted this and changed the linting.